docs/public-networks/reference/engine-api/objects.md

```diff
@@ -9,6 +9,15 @@ tags:
 
 The following objects are parameters for or returned by the [Engine API methods](index.md).
 
+## Blob and proof object
+
+Returned by [`engine_getBlobsV1`](index.md#engine_getblobsv1).
+
+| Key | Type | Value |
+| --- | :-: | --- |
+| `blob` | _Data_, 131072 Bytes | An SSZ-encoded `Blob` as defined in [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844). |
```

Comment:
```suggestion
| `blob` | _Data_, 131072 Bytes | An SSZ-encoded `blob` as defined in [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844). |
```

Reply:
proper noun for blob?

Reply:
If proper noun, then consider dropping the ``

Reply:
The type alias is capitalized and code formatted according to the linked EIP.